packages/js/product-editor/src/components/attribute-list-item/attribute-list-item.scss

```diff
@@ -37,4 +37,26 @@
 		justify-content: end;
 		gap: $gap-smaller;
 	}
+
+	&__actions-tooltip.components-tooltip.components-popover
+		.components-popover__content {
+		width: 100%;
+		max-width: 150px;
+		white-space: normal;
+	}
+
+	&__actions-icon-wrapper {
+		display: flex;
+		align-items: center;
+		position: relative;
+
+		&-icon {
+			color: $gray-600;
+		}
+		&-help-icon {
+			position: absolute;
+			right: -2px;
+			bottom: 0px;
```

Comment:
Made absolute so that the help icon overlaps the other icon a little bit.
They are also different colours which is why I treated them as separate icons instead of one.